app/src/main/java/org/mozilla/fenix/settings/creditcards/view/CreditCardsManagementView.kt

```diff
@@ -41,6 +41,9 @@ class CreditCardsManagementView(
         credit_cards_list.isVisible = state.creditCards.isNotEmpty()
 
         creditCardsAdapter.submitList(state.creditCards)
+        if (!state.isLoading && state.creditCards.isEmpty()) {
+            interactor.onCreditCardListEmptied()
+        }
```

Comment:
Let's add a new line before these kind of if statement blocks to provide better code separation to make it easier to read.

```suggestion

        if (!state.isLoading && state.creditCards.isEmpty()) {
            interactor.onCreditCardListEmptied()
        }
```